src/raftstore/store/fsm/peer.rs

```diff
@@ -345,15 +345,15 @@ impl<'a, T: Transport, C: PdClient> PeerFsmDelegate<'a, T, C> {
         // If there is no merge lock for that key, insert one to let target peer know `PrepareMerge`
         // is already executed.
         let mut meta = self.ctx.store_meta.lock().unwrap();
-        let (exist_version, ready_to_merge) =
+        let (exist_version, ready_to_handle) =
```

Comment:
I think `ready_to_merge` is more clear and specific.

Reply:
We already have `ready_to_merge` in `apply.rs`, it is confusing if same here. And `ready_to_merge` is more like applying merge rather than handling merge apply result

Reply:
It's name `ready_to_merge` on purpose. Because it's basically the same as apply, giving them the same name can help better understand the code. `ready_to_apply_merge` is about applying merge, but not `ready_to_merge`.

Reply:
okay, I have use `ready_to_handle` in other places, it will rename them later